service/src/main/java/com/epam/esm/validator/impl/CertificateValidatorImpl.java

```diff
@@ -38,13 +46,13 @@ public CertificateValidatorImpl(TagValidator tagValidator) {
     @Override
     public Map<String, String> validateForUpdate(Certificate certificate) {
         Map<String, String> result = new HashMap<>();
-        if (!isEmpty(certificate.getName())) {
+        if (!StringUtils.isBlank(certificate.getName())) {
```

Comment:
Best practice is trying to avoid negations in conditions where possible, so use isNotBlank please.

Reply:
StringUtils.isNonEmpty() perform also trim action. So you can chose what is more preferable for you, isEmpty (with trim action) or isBlank(without trim action)